R/util-func-namespace.R

```diff
@@ -21,5 +21,7 @@ GetStrFunctionIfNamespaced <- function(strFunction) {
     fn <- fn_pieces[[2]]
     return(rlang::as_function(fn, env = getNamespace(pkg)))
   }
-  return(strFunction)
+  else {
+    return(rlang::as_function(strFunction))
+  }
```

Comment:
```suggestion
  return(rlang::as_function(strFunction))
```
Since you've already `return()`ed the other case, you don't even need `else` here.